bin/cron/deploy_to_staging_next

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env ruby
+
+# A script that performs a merge of the test branch into the staging-next branch.
+#
+# NOTE: This script assumes it is only run from the test environment.
+
+require_relative '../../deployment'
+require 'cdo/chat_client'
+require 'cdo/developers_topic'
+require 'cdo/infra_test_topic'
+require 'cdo/github'
+require 'cdo/only_one'
+
+# Creates a new branch with the given name from the given commit on the test server
+# (without checking it out) and pushes it to GitHub.
+# @param [String] branch_name The name of the branch to create.
+# @param [String] commit The sha of the commit to base the branch at.
+def create_branch_from_commit(branch_name, commit)
+  system(
+    [
+      'cd ~/test',
+      "git branch --no-track #{branch_name} #{commit}",
+      "git push origin #{branch_name}"
+    ].join(' && ')
+  )
+end
+
+# Deletes a branch with the given name, both locally and on GitHub.
+# @param [String] branch_name The name of the branch to delete.
+def delete_branch(branch_name)
+  system(
+    [
+      'cd ~/test',
+      'git branch -D branch_name',
+      'git push origin --delete branch_name'
+    ].join(' && ')
+  )
+end
+
+# @return [Boolean] Whether the Slack#developers topic specifies "DTSN: yes".
+def slack_permission?
+  # TODO: Once DTSN is working, respect it here.
```

Comment:
What's the timeline/owner for getting DTSN working?

Reply:
@ewjordan is working on it in https://github.com/code-dot-org/code-dot-org/pull/19366.  I don't know what timeline he has it mind, but last time I looked it was pretty far along - I wouldn't be surprised if it got set up today.

Reply:
This is merged, and will hopefully work whenever staging-next rebuilds.